pkg/client/clientset_generated/internalclientset/typed/extensions/unversioned/generated_expansion.go

```diff
@@ -24,6 +24,8 @@ type IngressExpansion interface{}
 
 type JobExpansion interface{}
 
+type PodSecurityPolicyExpansion interface{}
```

Comment:
I seemed to have to add these manually or it wouldn't generate correctly.  Either I'm doing it wrong (likely) or possibly running into something like https://github.com/kubernetes/kubernetes/issues/20549